rpcs3/Emu/Cell/lv2/sys_net/lv2_socket.h

```diff
@@ -79,7 +85,7 @@ class lv2_socket
 
 public:
 	virtual std::tuple<bool, s32, std::shared_ptr<lv2_socket>, sys_net_sockaddr> accept(bool is_lock = true) = 0;
-	virtual s32 bind(const sys_net_sockaddr& addr) = 0;
+	virtual s32 bind(const sys_net_sockaddr& addr)                                                           = 0;
```

Comment:
are you serious???

Reply:
Look at that beautiful alignment though :D